src/main/java/com/mabl/integration/jenkins/MablStepDeploymentRunner.java

```diff
@@ -242,7 +242,9 @@ private void printAllJourneyExecutionStatuses(final ExecutionResult result) {
         for (ExecutionResult.ExecutionSummary summary : result.executions) {
             outputStream.printf("  Plan [%s] is [%s]%n", safePlanName(summary), summary.status);
             for (ExecutionResult.JourneyExecutionResult journeyResult : summary.journeyExecutions) {
-                outputStream.printf("  Journey [%s] is [%s]%n", safeJourneyName(summary, journeyResult.id), journeyResult.status);
+                String journeyFormat = String.format("    Journey [%s] is [%s]", safeJourneyName(summary, journeyResult.id), journeyResult.status);
+                outputStream.printf(journeyResult.status.equalsIgnoreCase("failed") ? String.format(journeyFormat + " at [%s]%n", journeyResult.appHref) :
```

Comment:
This feels a bit messy with multiple format statements and concatenations in a ternary operator all in one line. Maybe break some more out into their own lines or a helper function? Maybe remove the concat in favor of just `String.format`

Reply:
@mikedecosta I broke down the ternary to their own lines. Maybe I'm not seeing it how to get rid of concat because I want to keep journeyFormat in one place.